pycue/opencue/wrappers/layer.py

```diff
@@ -282,6 +282,12 @@ def range(self):
         @return: Layer frame range"""
         return self.data.range
 
+    def chunkSize(self):
+        """Returns the chunk size used while processing the layer.
+        @rtype:  int
+        @return: the chunks size"""
```

Comment:
nit: Could be a bit more descriptive here, maybe something like "the number of frames per task".

Reply:
done

Reply:
Thanks!